### PR TITLE
Sefcom

### PR DESCRIPTION
**Pull request by bngabonziza**
Debugging Qemu using GDB

### CODE REVIEW

**Comment by pascal-brand-st-dev** [comment]
https://github.com/OP-TEE/optee_os/blob/master/documentation/debug.md already contains steps to install and use gdb to debug on qemu. Could you update accordingly this file instead?

Thanks,
Pascal.

File: documentation/debug.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ set print pretty on
 
 define optee
 	handle SIGTRAP noprint nostop pass
-	symbol-file $HOME/devel/qemu_optee/optee_os/out/arm-plat-vexpress/core/tee.elf
+	symbol-file ~/devel/qemu_optee/optee_os/out/arm-plat-vexpress/core/tee.elf
```
Comment (jbech-linaro) [inline documentation/debug.md]: This particular change doesn't add much value.

File: documentation/debug.md [hunk 2]
```diff
@@ -97,8 +97,8 @@ terminal]
 Now it is time to set the breakpoints. For example
 
 ```
-(gdb) b plat_tee_entry
-Breakpoint 1 at 0x7df104d6: file core/arch/arm/plat-vexpress/plat_tee_func.c, line 42.
+(gdb) b tee_entry
```
Comment (jbech-linaro) [inline documentation/debug.md]: Since plat_tee_entry isn't existing any longer, I think it's a good idea to update this (and the ones below).

**Comment by jbech-linaro** [comment]
@bngabonziza could you please send an updated patch based on my last comments?

**Comment by pascal-brand-st-dev** [comment]
@bngabonziza Please finalize the contribution, as detailed in https://github.com/OP-TEE/optee_os/blob/master/documentation/github.md#finalizing-the-contribution

We'll close the pull-request if there is no activity by next week.

**Comment by pascal-brand-st** [comment]
Closing as no activity

**Pull request closed without merging**